modules/weko-itemtypes-ui/weko_itemtypes_ui/views.py

```diff
@@ -149,6 +161,16 @@ def get_property_list(property_id=0):
     lang = request.values.get('lang')
 
     props = ItemTypeProps.get_records([])
+
```

Comment:
@weko3-dev23 
Below source code is duplicated with source code at line 141 -> Should be create common function.

Reply:
Thank for your comment.
It had been fix at last rev.